### PR TITLE
Add pipeline tests

### PR DESCRIPTION
**Pull request by ktagowski**

### CODE REVIEW

File: tests/test_text_classification.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    task = FlairTextClassification(RESULTS_PATH, task_train_kwargs={"max_epochs": 1})
+    model = FlairModel(embedding, task)
+    evaluator = TextClassificationEvaluator()
+    pipeline = StandardPipeline(dataset, data_loader, transformation, model, evaluator)
+    return pipeline
+
+
+def test_text_classification_pipeline(text_classification_pipeline: StandardPipeline) -> None:
+    result = text_classification_pipeline.run()
+    assert 0 < result[0]["accuracy"] < 1
```
Comment (laugustyniak) [inline tests/test_text_classification.py:40]: `f1` please :)

Comment (riomus) [reply tests/test_text_classification.py]: i ustawmy tez minimum takie jak osiaga model aktualnie

File: tests/test_text_classification.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    return pipeline
+
+
+def test_text_classification_pipeline(
+    text_classification_pipeline: StandardPipeline[
+        str, datasets.DatasetDict, Corpus, Dict[str, np.ndarray], List[Any]
+    ]
+) -> None:
+    result = text_classification_pipeline.run()
+    assert 0 <= result[1]["f1"] <= 1
```
Comment (riomus) [inline tests/test_text_classification.py:53]: zamiast zero dajmy realne minimum

Comment (ktagowski) [reply tests/test_text_classification.py:53]: No ale niestety 0 to także realny case, musielibyśmy ręcznie zrobić subsampling

Comment (ktagowski) [reply tests/test_text_classification.py:53]: https://github.com/CLARIN-PL/embeddings/pull/17/checks?check_run_id=2655260377

File: tests/test_sequence_tagging.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    dataset = HuggingFaceDataset("clarin-pl/nkjp-pos")
+    data_loader = HuggingFaceDataLoader()
+    transformation = Transformations(
+        [
+            ColumnCorpusTransformation("tokens", "pos_tags"),
+            DownsampleFlairCorpusTransformation(percentage=0.0025),
+        ]
+    )
+    embedding = FlairTransformerWordEmbedding("allegro/herbert-base-cased")
+    task = SequenceTagging(RESULTS_PATH, hidden_size=256, task_train_kwargs={"max_epochs": 1})
```
Comment (ktagowski) [inline tests/test_sequence_tagging.py:35]: I think that these paths should be temporal. I wIll change it before merge

**Review by laugustyniak — APPROVED** [review]
I like the idea of then operation 😁

**Pull request merged**